test/Index/roles.swift

```diff
@@ -175,9 +175,9 @@ class AClass {
 }
 
 let _ = AClass.foo
-// CHECK: [[@LINE-1]]:16 | instance-method/Swift | foo() | [[AClass_foo_USR]] | Ref | rel: 0
+// CHECK: [[@LINE-1]]:16 | instance-method/Swift | foo() | [[AClass_foo_USR]] | Ref,Call | rel: 0
 let _ = AClass(x: 1).foo
-// CHECK: [[@LINE-1]]:22 | instance-method/Swift | foo() | [[AClass_foo_USR]] | Ref | rel: 0
+// CHECK: [[@LINE-1]]:22 | instance-method/Swift | foo() | [[AClass_foo_USR]] | Ref,Call | rel: 0
```

Comment:
Marking these as "call sites" is inaccurate, could you correct this?

Reply:
This test passes with no changes in the latest version of the commit.